project/SparkBuild.scala

```diff
@@ -464,7 +464,20 @@ object DockerIntegrationTests {
  */
 object DependencyOverrides {
   lazy val settings = Seq(
-    dependencyOverrides += "com.google.guava" % "guava" % "14.0.1")
+    dependencyOverrides += "com.google.guava" % "guava" % "14.0.1",
+    dependencyOverrides += "com.fasterxml.jackson.core" % "jackson-annotations" % "2.6.7",
+    dependencyOverrides += "com.fasterxml.jackson.core" % "jackson-core" % "2.6.7",
+    dependencyOverrides += "com.fasterxml.jackson.core" % "jackson-module-jaxb-annotations" % "2.6.7",
+    dependencyOverrides += "com.fasterxml.jackson.core" % "jackson-databind" % "2.6.7")
```

Comment:
These look coming from `jackson-jaxrs-json-provider` where it looks the resolution is different between Maven and SBT. I had to manually override and exclude.